tests/Assetic/Test/Extension/Twig/TwigResourceTest.php

```diff
@@ -24,9 +24,9 @@ protected function setUp()
 
     public function testInvalidTemplateNameGetContent()
     {
-        $loader = $this->getMock('Twig_LoaderInterface');
+        $loader = $this->getMock('Twig_LoaderInterface', array('getSource', 'getSourceContext', 'getCacheKey', 'isFresh', 'exists'));
```

Comment:
no, we should not force the separation between both, as Twig precisely ensures than new APIs are available in 1.x too (and we want to run on 1.x without deprecation warnings, meanig using the new APIs already when they are available, giving us Twig 2 support for free)